docker-compose.regtest.yml

```
@@ -1,20 +1,60 @@
+    container_name: pgnbxplorer
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    volumes:
+      - type: tmpfs
+        target: /var/lib/postgresql/data
+  nbxplorer:
+    restart: unless-stopped
+    container_name: nbxplorer
+    ports: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)